db/migrate/20180214150849_add_locked_to_podcasts.rb

```diff
@@ -0,0 +1,15 @@
+class AddLockedToPodcasts < ActiveRecord::Migration
```

Comment:
Had to add this to undo the prior migration as it was already deployed to staging by the PR test in CICD.